Classes/Command/UpdateChildRelationCommand.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+namespace Pixelant\PxaProductManager\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class UpdateChildRelationCommand extends Command
+{
+    protected const PRODUCT_TABLE = 'tx_pxaproductmanager_domain_model_product';
+    protected const ATTRIBUTEVALUE_TABLE = 'tx_pxaproductmanager_domain_model_attributevalue';
+    protected const RELATION_INDEX_TABLE = 'tx_pxaproductmanager_relation_inheritance_index';
+
+    /**
+     * Configure the command by defining the name, options and arguments.
+     */
+    protected function configure(): void
+    {
+        $this->setDescription('Update child relations table.');
```

Comment:
I think it's good to call it the "relation inheritance index" so we don't mix up the terminology.